make/utils.cmake

```diff
@@ -235,6 +235,30 @@ function (check_avx512_processor_and_compiler_support out)
   set(${out} ${proc_found_avx512} PARENT_SCOPE)
 endfunction (check_avx512_processor_and_compiler_support)
 
+function(check_intel_pt_support out)
```

Comment:
This check for PT, which is used at runtime, is being performed at config/build time.  Please add a comment to callers to *not* use this to make any decisions on building exported modules -- someone should be able to build a PT-capable binary on any system with the right headers, so this should not be used to turn off building non-tests.

Reply:
Done.